### DISCUSSION TITLE
Status of RDMA support

### DISCUSSION THREAD

**Discussion by nlgranger** [nfs-ganesha/nfs-ganesha#1203] [Q&A]
There have been a few commit about RDMA earlier this year and the support is enabled by default in Cmake.
Does this mean it is supported again ? 

https://github.com/nfs-ganesha/nfs-ganesha/blob/f503dc4067aa1ef23f878635f18e8dfbc5786ced/src/CMakeLists.txt#L386

**Comment by dang** [comment]
It works, for some definitions of "works".  Let's call it early access, rather than full support.  More work on it is incoming, hopefully soon.

**Comment by nlgranger** [comment]
All right, thanks for the clarification. I'll keep an eye on future updates.

**Comment by Manabreaker** [comment]
Are there any updates since? 🙃

**Comment by ffilz** [comment]
Ganesha V10.0 will have additional RDMA support including back channel.
